packages/govuk-frontend/src/govuk/components/accordion/accordion.mjs

```diff
@@ -113,7 +114,11 @@ export class Accordion {
    * @param {AccordionConfig} [config] - Accordion config
    */
   constructor ($module, config) {
-    if (!($module instanceof HTMLElement) || !document.body.classList.contains('govuk-frontend-supported')) {
+    if (!document.body.classList.contains('govuk-frontend-supported')) {
+      throw new GOVUKFrontendNotSupportedError()
```

Comment:
Just a random thought

Similar to how `TypeError` isn't `NotCorrectTypeError`, should we flip out the `Not`?

```suggestion
      throw new GOVUKFrontendSupportError()
```

Reply:
Fair, and that makes it shorter, I'll rename 😊